hw/index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Cubes</title>
+    <link rel="stylesheet" href="./src/css/style.css" type="text/css">
+</head>
+
+<body>
+    <section class="weatherWidget">
+        <div class="block first">
+            <picture class="weather">
+                <source srcset="src/img/CRAlt.svg" type="image/svg+xml">
+                <img src="src/img/CRAlt.svg" alt="rainy weather">
+            </picture>
+            <p class="forecast">Chicago / 23&#176;</p>
+        </div>
+        <div class="block second">
+            <picture class="weather">
+                <source srcset="src/img/Fog.svg" type="image/svg+xml">
+                <img src="src/img/Fog.svg" alt="foggy weather">
+            </picture>
+            <p class="forecast">Seatle / 34&#176;</p>
+        </div>
+        <div class="block third">
+            <picture class="weather">
+                <source srcset="src/img/CS.svg" type="image/svg+xml">
+                <img src="src/img/CS.svg" alt="partly cloudy weather">
+            </picture>
+            <p class="forecast">Brooklyn / 55&#176;</p>
+        </div>
+        <div class="block fourth">
+            <picture class="weather">
+                <source srcset="src/img/CSS.svg" type="image/svg+xml">
+                <img src="src/img/CSS.svg" alt="snowy weather">
+            </picture>
+            <p class="forecast">Portland / 27&#176;</p>
+        </div>
+        <div class="block fifth">
+            <picture class="weather">
+                <source srcset="src/img/CWM.svg" type="image/svg+xml">
+                <img src="src/img/CWM.svg" alt="windy weather">
+            </picture>
+            <ul class=inWeek>
```

Comment:
It is preferable to use kebab-case rather than camelCase in CSS, so I would suggest changing the name to "in-week". Also, if we are following the BEM naming methodology (https://ru.bem.info/methodology/quick-start/), the class names should be as following:
```
<div class="weather-widget__block weather-widget__block_fifth>
    <div class="weather-widget__weather></div>
    <div class="weather-widget__in-week></div>
</div>
```
The same naming principle shall be applied to other blocks as well. We are using "weather-widget" as part of all classes' names in the example above because all these classes are intended to be used only inside the "weather-widget" tag and nowhere else.